src/webview/columns/data-column.tsx

```diff
@@ -47,11 +47,11 @@ export class DataColumn implements ColumnContribution {
                 const isLast = i + 1n >= range.endAddress;
                 const style: React.CSSProperties | undefined = isLast ? undefined : this.byteGroupStyle;
 
-                groups.push(<span className='byte-group' style={style} key={i.toString(16)}>{words}</span>);
+                groups.push(<span className='byte-group hoverable' data-column='data' style={style} key={i.toString(16)}>{words}</span>);
```

Comment:
Do you think it might make sense to apply additional hover feedback/highlighting on the span? Since we might have several words per group it could really make it more obvious which part we actually show hover information for.

Reply:
How about a dotted underline like this?
![image](https://github.com/eclipse-cdt-cloud/vscode-memory-inspector/assets/20653241/3a82842f-6218-4b19-acde-ea456c48c698)

Reply:
It is very subtle but I like it, thank you!